inference-engine/src/plugin_api/threading/ie_tbb_streams_executor.hpp

```diff
@@ -11,6 +11,7 @@
 #include "ie_parallel.hpp"
 #include "threading/ie_istreams_executor.hpp"
 
+#if ((IE_THREAD == IE_THREAD_TBB) || (IE_THREAD == IE_THREAD_TBB_AUTO))
```

Comment:
why compilation is failed here? I don't see any TBB specifics in this file

Reply:
From the comments below and our compilation message, it seems TbbStreamExecutor will use TBB when we set THREADING=SEQ in windows.